src/store/metadata/stub/content/editions/versions/schools-and-nursery-schools-rated_2023-05.json

```diff
@@ -0,0 +1,88 @@
+{
+    "@context": "https://staging.idpd.uk/#ns",
+    "@id": "https://staging.idpd.uk/datasets/schools-and-nursery-schools-rated/editions/2023-05/versions",
+    "@type": "hydra:Collection",
+    "title": "Schools and nursery schools rated good or outstanding",
+    "versions": [
+        {
+            "@type": [
+                "dcat:Distribution",
+                "csvw:Table"
+            ],
+            "@id": "https://staging.idpd.uk/datasets/schools-and-nursery-schools-rated/editions/2023-05/versions/1",
+            "identifier": "1",
+            "issued": "2023-05-16T13:36:00Z",
+            "title": "Schools and nursery schools rated good or outstanding",
+            "summary": "Percentage of state-funded provided for schools that are rated",
```

Comment:
Change this to 'Percentage of state-funded schools that are rated good or outstanding'

Reply:
completed